scripts/resource-creation/create-project-namespace-deployments.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
+
+# Still to do
+# Persist project name to id map (in case of failure)
+# Leave more gaps between creating projects (crashes my DO 2 out of 3 times)
+# Convert YAML output to $RANCHER_API/v1/management.cattle.io.clusters/$CLUSTER_ID?action=apply request (needs multi line `\n`ing) 
+# Clean up resources (would need to delete projects and their namespaces)
+
+# Optional args
+PROJECTS=300
+NAMESPACES=1000
+DEPLOYMENTS=3000
+
+# Require args
+# Note key must NOT be scoped
+CATTLE_ACCESS_KEY=
```

Comment:
Could we map the values to env var, so we can avoid to worry about changing the file?

Reply:
The `getopts` section below allows them to be supplied when executing the script `./.h -a <access key>` etc